tests/report_tests/plugin_tests/logs.py

```diff
@@ -77,7 +82,7 @@ def test_journal_tailed_and_linked(self):
         assert os.path.islink(journ), "Journal in sos_commands/logs is not a symlink"
 
     def test_string_not_in_manifest(self):
-        # we don't want truncated collections appearing in the strings section
+        # we don't want truncated journals appearing in the strings section
         # of the manifest for the plugin
         manifest = self.get_plugin_manifest('logs')
-        self.assertFalse(manifest['strings'])
+        self.assertFalse(any('journalctl' in _ms for _ms in manifest['strings']))
```

Comment:
This does work well even if manifest does not contain `strings` section..? (I need to test that).

Reply:
`strings` will always be a section in the manifest, but this does work even if it is unpopulated.